docs/developer-docs/latest/getting-started/quick-start.md

```diff
@@ -1,150 +1,488 @@
+1. Click on **+ Add New Categories**.
+2. Type `French Food` in the **Name** field.
+3. On the right, in the **Restaurants (0)** dropdown, select `Biscotte Restaurant`.
+4. Click **Save**.
+5. Go back to _Collection types > Categories_, then click again on **+ Add New Categories**.  
+6. Type `Brunch` in the **Name** field, then click **Save**.
 
-## 6. Add categories to the "Category" Content Type
+![GIF: Add Categories](../assets/quick-start-guide/qsg-handson-categories.gif)
 
-Navigate to [**COLLECTION TYPES** - **Categories**](http://localhost:1337/admin/plugins/content-manager/collectionType/application::category.category).
+The `French Food` and `Brunch` categories are now listed in the [_Collection types > Categories_](http://localhost:1337/admin/plugins/content-manager/collectionType/application::category.category) view.
 
-- Click on **+ Add New Categories** button. Type `French Food` in the **Name** field. Select `Biscotte Restaurant`, on the right in the **Restaurant (0)** dropdown.
-- Click **Save**.
+#### Add a Category to a Restaurant
 
-Navigate back to [**COLLECTION TYPES** - **Categories**](http://localhost:1337/admin/plugins/content-manager/collectionType/application::category.category). You will see the **French Food** category listed in the entries.
+Go to [_Collection types > Restaurants_](http://localhost:1337/admin/plugins/content-manager/collectionType/application::restaurant.restaurant) in the left-hand menu, and click on `Biscotte Restaurant`.
 
-- Click on **+ Add New Categories** button. Type `Brunch` in the **Category** field. **DO NOT ADD** `Biscotte Restaurant` to the `Restaurants` dropdown on the right.
-- Click **Save**.
+In the right sidebar, under **Categories (1)**, click on **Add an item…** and select `Brunch`.  Click **Save**.
 
-You will see the **Brunch** category listed in the entries.
+### Step 3: Set Roles & Permissions
 
-Navigate back to [**COLLECTION TYPES** - **Restaurants**](http://localhost:1337/admin/plugins/content-manager/collectionType/application::restaurant.restaurant).
+1. Click on _General > Settings_ at the bottom of the left-hand menu.
+2. Under _Users & Permissions Plugin_, choose [**Roles**](http://localhost:1337/admin/settings/users-permissions/roles).
+3. Click the **Public** role.
+4. Scroll down under **Permissions**.
+5. In the **Application** tab, find **Restaurant**. 
+6. Click the checkboxes next to **find** and **findone**.
+7. Repeat with **Category**: click the checkboxes next to **find** and **findone**.
+8. Finally, click **Save**.
 
-- Click on `Biscotte Restaurant`.
-- On the right, under **Categories(1)**, `select` the `Add an item...`, and add **Brunch** as a category for this restaurant, and click the **Save** button.
+![Screenshot: Public Role in Users & Permissions plugin](../assets/quick-start-guide/qsg-handson-part2-04-roles.png)
```

Comment:
This is not the right image, it's a screenshot of the relation and not the permissions.

Reply:
Fixed, thanks!

Reply:
🤦  Not sure what happened, was supposed to be updated with [this commit](https://github.com/strapi/documentation/pull/288/commits/13dfaa564364f85334aaf1eeacb208fe46831cbb?short_path=1ce7981#diff-1ce7981e6b01a50e7dfdfcc1a91fba1e58baaf7b9731f6b9b8f01e351d6a5c44) 🤷 
Should be good [now](https://github.com/strapi/documentation/pull/288/commits/9bde13c78fe3634a31bf90e26c339dcbc603e0a8?short_path=87b726f#diff-87b726f260430d4c61df3a73fd394288925822e0bfc6179f0abd17a4f78abb9f)